app/jobs/import_versions_job.rb

```diff
@@ -94,6 +102,7 @@ def import_record(record)
     end
 
     if @import.skip_unchanged_versions? && version_changed?(version)
+      log(object: version, operation: :skipped, row: row)
```

Comment:
Might be helpful to include something clarifying here that we skipped it because it was identical to the version preceding it, rather than because we already have a copy of it (that skip happens up on line 85).